src/components/node/NodeModelSelect.vue

```diff
@@ -137,16 +137,23 @@ export default Vue.extend({
     });
 
     /**
-     * Triggers when parameter is changed.
+     * Triggers when parameter selection is changed.
      */
-    const selectionChange = () => {
+    const paramSelectionChange = () => {
       state.node.params.forEach(
         (param: ModelParameter) =>
           (param.state.visible = state.visibleParams.includes(param.idx))
       );
       state.node.nodeChanges();
     };
 
+    /**
+     * Triggers when node model is changed.
+     */
+    const modelChange = () => {
+      update();
+    };
+
```

Comment:
I thought a self-explanation function name would be useful.

Reply:
If the name `update` is not self-explanatory enough (which can certainly be justified), it would be the cleaner solution to rename `update completely (to something like `updateOnModelChange`), I guess?